spotlight/src/main/java/com/takusemba/spotlight/SimpleTarget.java

```diff
@@ -11,147 +11,145 @@
-         * @return This Builder
-         */
-        public Builder setTitle(CharSequence title) {
-            this.title = title;
-            return this;
-        }
-
-        /**
-         * Set the description text shown on Spotlight
-         *
-         * @param description title shown on Spotlight
-         * @return This Builder
-         */
-        public Builder setDescription(CharSequence description) {
-            this.description = description;
-            return this;
-        }
-
-        /**
-         * Create the {@link SimpleTarget}
-         *
-         * @return the created SimpleTarget
-         */
-        @Override
-        public SimpleTarget build() {
-            if (getContext() == null) {
-                throw new RuntimeException("context is null");
-            }
-            View view = getContext().getLayoutInflater().inflate(R.layout.layout_spotlight, null);
-            ((TextView) view.findViewById(R.id.title)).setText(title);
-            ((TextView) view.findViewById(R.id.description)).setText(description);
-            PointF point = new PointF(startX, startY);
-            calculatePosition(point, radius, view);
-            return new SimpleTarget(point, radius, view, listener);
-        }
-
-        /**
-         * calculate the position of title and description based off of where the spotlight reveals
-         */
-        private void calculatePosition(final PointF point, final float radius, View spotlightView) {
-            float[] areas = new float[2];
-            Point screenSize = new Point();
-            ((WindowManager) spotlightView.getContext()
-                    .getSystemService(Context.WINDOW_SERVICE)).getDefaultDisplay().getSize(screenSize);
-
-            areas[ABOVE_SPOTLIGHT] = point.y / screenSize.y;
-            areas[BELOW_SPOTLIGHT] = (screenSize.y - point.y) / screenSize.y;
-
-            int largest;
-            if (areas[ABOVE_SPOTLIGHT] > areas[BELOW_SPOTLIGHT]) {
-                largest = ABOVE_SPOTLIGHT;
-            } else {
-                largest = BELOW_SPOTLIGHT;
-            }
-
-            final LinearLayout layout = spotlightView.findViewById(R.id.container);
-            layout.setPadding(100, 0, 100, 0);
-            switch (largest) {
-                case ABOVE_SPOTLIGHT:
-                    spotlightView.getViewTreeObserver()
-                            .addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
-                                @Override
-                                public void onGlobalLayout() {
-                                    layout.setY(point.y - radius - 100 - layout.getHeight());
-                                }
-                            });
-                    break;
-                case BELOW_SPOTLIGHT:
-                    layout.setY((int) (point.y + radius + 100));
-                    break;
-            }
-        }
-    }
+	private View view;
```

Comment:
it seems like the whole indent had been changed. Im using `default` code style in Android Studio. Is it possible not to make these changes